src/main/com/fulcrologic/fulcro/algorithms/form_state.cljc

```diff
@@ -47,15 +47,15 @@
 (defsc FormConfig
   "A component supporting normalization of form state configuration. Use Fulcro Inspect for viewing that data.
   Rendering isn't supported on this component so it will work with React Native.
-  can also render the form config, if that is useful to you."
+  Can also render the form config, if that is useful to you."
```

Comment:
Actually, this sentence isn't even true anymore.  This function used to render things, but it doesn't anymore so it can be used in React native...it's just for normalization.

Reply:
Then, do you suggest that we shall elaborate based on that use-case?